src/App.js

```diff
@@ -78,7 +78,7 @@ function App() {
     return unsubscribe;
   }, [dispatch]);
 
-  if (isLoading) return <Loader />;
+  // if (isLoading) return <Loader />;
```

Comment:
Why is this commented out?

Reply:
This will cause a weird spinner on the app when you refresh. We can completely remove it in debugging phase.